crates/aiken-lang/src/builder.rs

```diff
@@ -542,6 +538,32 @@ pub fn list_access_to_uplc(
             .into(),
         }
     } else if names.is_empty() {
+        // Maybe check list is actually empty or should we leave that to when .. is only
+        // this would replace term.into() if we decide to
+        // body: choose_list(
+        //     apply_wrap(
+        //         Term::Builtin(DefaultFunction::TailList).force_wrap(),
+        //         Term::Var(Name {
+        //             text: format!(
+        //                 "tail_index_{}_{}",
+        //                 current_index, id_list[current_index]
+        //             ),
+        //             unique: 0.into(),
+        //         }),
+        //     ),
+        //     term,
+        //     apply_wrap(
+        //         apply_wrap(
+        //             Term::Builtin(DefaultFunction::Trace).force_wrap(),
+        //             Term::Constant(UplcConstant::String(
+        //                 "List contains more items".to_string(),
+        //             )),
+        //         ),
+        //         Term::Delay(Term::Error.into()),
+        //     )
+        //     .force_wrap(),
+        // )
+        // .into(),
```

Comment:
We need to have a conversation about commented-out code in source files guys 😬 @MicroProofs @rvcas

Reply:
I'm not personally a fan but sometimes I do it when I might want to the code for later. Not exactly sure what Kasey wants to do with this one